simiyu1/JavaDevsNairobi
# #156187450 add network utility package

#### What does this PR do?
 - Add a util package holding the NetworkConnectivity class

#### Description of Task to be completed?
 - create package utils
 - create class NetworkConnectivityManager to check connection
 - add permissions check network state

#### How should this be manually tested?
 - Clone this repo and open it on your IDE (preferably Android Studio)
 - Run the code on an emulator with no internet
 - A snackbar is displayed informing you of the connection status
 - Swiping down will attempt a new connection

#### What are the relevant pivotal tracker stories?
 - [#156187450](https://www.pivotaltracker.com/story/show/156187450)

#### Screenshots
![image](https://user-images.githubusercontent.com/17185082/47060965-3eb52480-d1d8-11e8-8af2-04221949c1a0.png)
![image](https://user-images.githubusercontent.com/17185082/47060989-5be9f300-d1d8-11e8-8d5c-fb58b218132f.png)
